examples/js/controls/TransformControls.js

```diff
@@ -1007,9 +1007,7 @@
 			var y = ( pointer.clientY - rect.top ) / rect.height;
 
 			pointerVector.set( ( x * 2 ) - 1, - ( y * 2 ) + 1, 0.5 );
-			pointerVector.unproject( camera );
-
-			ray.set( camPosition, pointerVector.sub( camPosition ).normalize() );
+			ray.setFromCamera( pointerVector, camera );
```

Comment:
Yes. There is a lot of clean up to do in this file but I don't want to conflict with #6795!